src/etl/ETLSource.h

```diff
@@ -466,9 +466,11 @@ class ETLLoadBalancer
 private:
     std::vector<std::unique_ptr<ETLSource>> sources_;
 
+    std::uint32_t downloadRanges_ = 16;
```

Comment:
Can we just keep this function local and pass it around?

Reply:
Not super easily, since we only access the configuration in the constructor. 

The other option would be to save the configuration as a member